tests/Avalonia.Input.UnitTests/Avalonia.Input.UnitTests.csproj

```diff
@@ -1,6 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
This project (and some other test project) doesn't seem to have been converted to the new project system, whereas other tests such as `Avalonia.Base.UnitTests` have been converted. Is there a reason for this?